### PR TITLE
Add contributing guidelines

### PR DESCRIPTION
**Pull request by tmorrell**
Adds some contributing guidelines and removes the top level call for maintainers. I grabbed a lot of this from the invenio software template, so please feel free to change/improve/modify.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+* Detailed steps to reproduce the bug.
+
+Fix Bugs
+~~~~~~~~
+
+Look through the GitHub issues for bugs. Anything tagged with "bug"
+is open to whoever wants to implement it.
+
+Implement Enhancements
+~~~~~~~~~~~~~~~~~~
```
Comment (yarikoptic) [inline CONTRIBUTING.rst:28]:
```suggestion
Implement Enhancements
~~~~~~~~~~~~~~~~~~~~~~
```

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+Look through the GitHub issues for enhancements. Anything tagged with "enhancement"
+is open to whoever wants to implement it.
+
+Implement Feature Requests
+~~~~~~~~~~~~~~~~~~
```
Comment (yarikoptic) [inline CONTRIBUTING.rst:34]:
```suggestion
Implement Feature Requests
~~~~~~~~~~~~~~~~~~~~~~~~
```

depending on settings might lead to errorring out if to be rendered by sphinx

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+      $ git push origin name-of-your-bugfix-or-feature
+
+5. Submit a pull request through the GitHub website.
+
+Pull Request Guidelines
+-----------------------
+
+Before you submit a pull request, check that it meets these guidelines:
+
+1. The pull request should not break tests. If you add a new feature, please add a test case.
```
Comment (yarikoptic) [inline CONTRIBUTING.rst:108]: FWIW it is even worth adding a test while fixing a bug so that bug doesn't come back

File: README.rst [hunk 4]
```diff
@@ -29,12 +29,6 @@ should not expect to see any major API changes soon.
 .. _semantic versioning: http://semver.org/
 
 
-Call for Maintainers
---------------------
-
-Unfortunately, the current maintainers lack the spare time to properly support citeproc-py at the moment. Even though there are not many bugs being reported, these are currently not being addressed. If you are an active user of this package, consider volunteering to help with maintainership. Please create a ticket to get the ball rolling. Thanks!
-
-
 Requirements
```
Comment (yarikoptic) [inline README.rst:32]: reference/link added CONTRIBUTING.rst here?

**Review by yarikoptic — COMMENTED** [review]
some initial comments. Thanks for crafting this one up!

**Comment by tmorrell** [comment]
Thanks @yarikoptic, I've incorporated your feedback. I've also turned on the GitHub PR requirements, review requirements, and test requirements. So please leave a review if the new wording looks good!

**Review by yarikoptic — APPROVED** [review]

**Comment by yarikoptic** [comment]
I approved, honors are yours @tmorrell to have last peak  (since was awhile) and merge . Thanks!

**Pull request merged**